### DISCUSSION TITLE
[bug]:

### DISCUSSION THREAD

**Discussion by ghost** [pmndrs/react-spring#1894] [Support] [answered]
### Which react-spring target are you using?

- [X] `@react-spring/web`
- [ ] `@react-spring/three`
- [ ] `@react-spring/native`
- [ ] `@react-spring/konva`
- [ ] `@react-spring/zdog`

### What version of react-spring are you using?

@react-spring/web

### What's Wrong?

Following the documentation, you cannot import Parallax components:

As shown in these videos: https://imgur.com/a/iclSSlb

### To Reproduce

1.) npm install @react-spring/web or react-spring
2.) import { Parallax, ParallaxLayer } from "@react-spring/parallax";

### Expected Behaviour

Imports Parallax components.

### Link to repo

https://imgur.com/a/iclSSlb

**Comment by joshuaellis** [comment]
That image doesn't load for me? Can you point me to the issue in the documentation & perhaps supply a code sandbox?

Thanks!

**Comment by ghost** [comment, marked as answer]
Never mind, I found that all you need to install the @react-spring/parallax package manually because it isn't included in @react-spring/web or react-spring install.

**Comment by joshuaellis** [comment]
Great to hear you solved your issue, I'm gonna move this to a discussion for future people.
